### ISSUE TITLE
Missing / in "Authenticating with GitHub Apps" 

### ISSUE THREAD

**Issue by JeromeBollinger** [github/docs#24343]
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/authenticating-as-a-github-app-installation

### What part(s) of the article would you like to see updated?

There is the following example:
```sh
curl --request POST \
--url "https://api.github.comapp/installations/INSTALLATION_ID/access_tokens" \
--header "Accept: application/vnd.github+json" \
--header "Authorization: Bearer JWT"\
--header "X-GitHub-Api-Version: 2022-11-28"
```
where the url is missing a "/"
It should be:  `--url "https://api.github.com/app/installations/INSTALLATION_ID/access_tokens" \`

### Additional information

This part seems to be templated as this is to be found in the code line 46 (content/apps/creating-github-apps/authenticating-with-a-github-app/authenticating-as-a-github-app-installation.md): 
`--url "{% data variables.product.api_url_pre %}meta" \`

**Comment by cmwilson21** [comment]
@JeromeBollinger Thanks so much for opening an issue! I'll triage this for the team to take a look :eyes:

**Comment by skedwards88** [comment]
Thank you for catching this! Your proposed fix to add a `/` after `{% data variables.product.api_url_pre %}` to line 46 in `content/apps/creating-github-apps/authenticating-with-a-github-app/authenticating-as-a-github-app-installation.md`) is exactly correct. You or anyone else is welcome to open a PR to make this fix.

**Comment by cmwilson21** [comment]
Closing this with the fix in #24414
